packages/common/reader/src/reader.ts

```diff
@@ -0,0 +1,905 @@
+import { Container } from '@blocksuite/affine/global/di';
+import type {
+  AttachmentBlockModel,
+  BookmarkBlockModel,
+  EmbedBlockModel,
+  ImageBlockModel,
+  TableBlockModel,
+} from '@blocksuite/affine/model';
+import { AffineSchemas } from '@blocksuite/affine/schemas';
+import { MarkdownAdapter } from '@blocksuite/affine/shared/adapters';
+import type { AffineTextAttributes } from '@blocksuite/affine/shared/types';
+import {
+  createYProxy,
+  type DeltaInsert,
+  type DraftModel,
+  Schema,
+  Transformer,
+  type TransformerMiddleware,
+  type YBlock,
+} from '@blocksuite/affine/store';
+import { uniq } from 'lodash-es';
+import {
+  Array as YArray,
+  type Doc as YDoc,
+  Map as YMap,
+  Text as YText,
+} from 'yjs';
+
+import { getStoreManager } from './bs-store';
+
+const blocksuiteSchema = new Schema();
+blocksuiteSchema.register([...AffineSchemas]);
```

Comment:
what's the reason of why we got `MarkdownAdapter` but still so many customized `generateXXXMarkdownPreview`? @Saul-Mirone @EYHN

Reply:
It came before we move blocksuite to affine. At that time, using blocksutie in affine is still kind of trouble some and @EYHN just built the converters from scratch.

`MarkdownAdapter` is imported after we move bs to affine and only used to handle some specific block flavours.

However, it's possible to use markdown adapter to parse all the blocks and remove all the custom generators here as we discussed in slack.